### ISSUE TITLE
Dependency to old version of plugin_platform_interface

### ISSUE THREAD

**Issue by abolourian** [rodydavis/flutter_midi#27]
flutter_midi depends on plugin_platform_interface 1.0.1 and that breaks the apps with dependency on version 2.0.0. An example is when I add "file_picker" library to flutter_piano, I get this error:
```Because file_picker >=3.0.0-nullsafety.3 depends on plugin_platform_interface ^2.0.0 and flutter_midi 1.0.2 depends on plugin_platform_interface ^1.0.1, file_picker >=3.0.0-nullsafety.3 is incompatible with flutter_midi 1.0.2.
And because no versions of flutter_midi match >1.0.2 <2.0.0, file_picker >=3.0.0-nullsafety.3 is incompatible with flutter_midi ^1.0.2.
So, because flutter_piano depends on both flutter_midi ^1.0.2 and file_picker ^3.0.0, version solving failed.
pub get failed (1; So, because flutter_piano depends on both flutter_midi ^1.0.2 and file_picker ^3.0.0, version solving failed.)```

**Comment by djfishe** [comment]
@rodydavis will you be issuing any updates for this?

**Comment by rodydavis** [comment]
fixed on version `1.1.0`. reopen if it is still an issue
